### DISCUSSION TITLE
Issues with char '…'

### DISCUSSION THREAD

**Discussion by CTassisF** [tweepy/tweepy#1822] [Questions] [answered]
I am seeing issues when trying to post the character '…' (or u'…').

When concatenating u'…' to a tweet I see:
'ascii' codec can't encode character u'\u2026' in position 0: ordinal not in range(128)

When concatenating '…' to a tweet I see:
'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)

I'm using Tweepy 3.3. This used to work with v3.0.

Can you please help me?
Thanks in advance.

**Comment by MechaCoder** [comment]
try useing "{}".format this solved the issue for me.

**Comment by CTassisF** [comment]
@MechaCoder Sorry, I did not understand. Can you please show me a working Python code example?

**Comment by MechaCoder** [comment]
yep sorry so the fix i had was to covert the whole sting into a normal python string using

string1 = "more"
string2 = "stuff"
Example = "{}{}".format(string1, string2)

so the Example would have the output of morestuff and convert both variables are converted to strings from unicode

hope this helps ^_^

**Comment by Harmon758** [comment, marked as answer]
`…` is not an ASCII character. You should use a Unicode encoding like UTF-8 instead.
This is an issue with handling encodings (with Python 2) rather than an issue with Tweepy.
